Orange/data/io.py

```diff
@@ -312,22 +328,25 @@ def data_table(self, data, headers=None):
                 #   1) delimited column names
                 #   2) -||- with type and flags prepended, separated by #,
                 #      e.g. d#sex,c#age,cC#IQ
-                _flags, names = zip(*[i.split(HEADER1_FLAG_SEP, 1) if HEADER1_FLAG_SEP in i else ('', i)
-                                      for i in headers[0]])
+                _flags, names = zip(*[
+                    i.split(HEADER1_FLAG_SEP, 1) if HEADER1_FLAG_SEP in i else (
+                    '', i)
+                    for i in headers[0]])
                 names = list(names)
             elif 2 == len(headers):
                 names, _flags = map(list, headers)
             else:
                 # Use heuristics for everything
                 names, _flags = [], []
-            types = [''.join(filter(str.isupper, flag)).lower() for flag in _flags]
+            types = [''.join(filter(str.isupper, flag)).lower() for flag in
+                     _flags]
```

Comment:
Could you at least break on `for`? When I was initiated, @astaric said we keep `--max-line-length` at about 120.